steampy/client.py

```diff
@@ -181,7 +184,11 @@ def accept_trade_offer(self, trade_offer_id: str) -> dict:
     def _fetch_trade_partner_id(self, trade_offer_id: str) -> dict:
         url = self._get_trade_offer_url(trade_offer_id)
         offer_response_text = self._session.get(url).text
-        return text_between(offer_response_text, "var g_ulTradePartnerSteamID = '", "';")
+        if "You have logged in from a new device. In order to protect the items" in offer_response_text:
+            trade_hold = False
```

Comment:
Bad practise, dont return Bool when it should return String. You can add Exception SevenDaysHoldException and raise it there.